arangod/RocksDBEngine/RocksDBOptionFeature.cpp

```diff
@@ -1526,6 +1527,37 @@ limited number of edge collections/shards/indexes.)");
   can open. Thus the option should only be enabled on deployments with a
   limited number of edge collections/shards/indexes.)");
 
+  options
+      ->addOption("--rocksdb.partition-files-for-vector-index",
+                  "If enabled, the index data for different vector "
+                  "indexes will end up in different .sst files.",
```

Comment:
Maybe change this to "Whether to use a separate .sst file for each vector index"?

Reply:
This was copy/pasted from other indexes (mdi, edge and persistent) which have the same option